docs/9.x/config.md

```diff
@@ -65,6 +65,18 @@ spec:
     # but the selected backend can be overridden if specified in the cluster configuration.
     # Supports values "vxlan", "aws-vpc", and "gce".
     flannelBackend: "vxlan"
+    # Enables Kubernetes resource encryption at rest.
+    # Rotating keys or disabling encryption requires manual configuration changes.
+    # See https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/ and
+    # https://github.com/kubernetes-sigs/aws-encryption-provider for more details.
+    encryptionProvider:
+      disabled: false
+      aws:
+        accountID: "1234567890"
+        keyID: "mrk-1234567890"
+        # region should be left unspecified when using a multi-region key. The region
+        # will be automatically selected from ec2 instance metadata.
+        region: "us-west-2"
```

Comment:
Given that a key ARN has a region in it (for all region specific keys), I'm wondering if we could simplify the UX here to only require users to put in a key ARN, from which the account/region/key id are derived internally by planet/gravity.

It isn't as configurable, but it also avoids extra knobs for twiddling.  Just a though -- you know these config paths better than I do.

Reply:
I addressed the comment in planet here https://github.com/gravitational/planet/pull/865#discussion_r721825796.
I separated the fields to allow the use of either a multi-region key or a single-region key. 

But 🤔 I guess we could still use a single ARN for a multi-region key. Multi-region keys share the same keyID and just have different regions. And we can distinguish multi-region keys because the keyID is prefixed with `mrk-`. We could parse the ARN and setup proper configuration if a multi-region key is detected.

The cluster configuration output might be a bit awkward since it would display the ARN with which ever arbitrary region was selected when setting up encryption.